README.md

```diff
@@ -1,46 +1,382 @@
+2. A PR must be merged by an SN engineer before a PR's code reviewers receive an award
+
+Code review approvals are more than welcome, but we can't guarantee awards for them because the work performed to approve a PR is unverifiable.
+
+<br>
+
+## Issue specification awards
+
+Issue specifiers will be awarded up to 10% of a PR award for issues resulting in a PR being merged by an SN engineer that closes the issue. In addition to being subject to PR award amounts and reductions, specification amounts are awarded on the basis of how much additional help and specification is required by other contributors.
+
+### Rules
+1. The issue must directly result in PR being merged by an SN engineer that closes the issue
+2. Issue specification award amounts are based on the final PR award amounts
+	- that is, they are subject to PR award code review reductions and priority multipliers
+3. Award amounts will be reduced on the basis of how much additional help and specification is required by other contributors
+4. Issue specifiers who can close their own issues with their own PRs are also eligible for this 10%
+    - e.g an issue tagged as `difficulty:hard` that is both specified and closed by a PR from the same contributor without changes requested awards 1.1m sats
+
+### Relative awards
+
+| circumstances                                                                                              | award |
+| ---------------------------------------------------------------------------------------------------------- | ----- |
+| issue doesn't require further help and/or specification from other contributors                            | 10%   |
+| issue requires little help and/or specification from other contributors                                    | 5%    |
+| issue requires more help and/or specification from other contributors than the issue specifier contributed | 1%    |
+| issue is vague and/or incomplete and must mostly be entirely specified by someone else                     | 0%    |
+
+For example: a specified issue that's tagged as `difficulty:hard`, doesn't require additional specification and disambiguation by other contributors, and results in PR being merged without changes requested awards the issue specifier 100k sats.
+
+<br>
+
+## Responsible disclosure of security or privacy vulnerability awards
+
+Awards for responsible disclosures are assessed on the basis of:
+
+1. the potential loss resulting from an exploit of the vulnerability
+2. the trivialness of exploiting the vulnerability
+3. the disclosure's detail
+
+Award amounts will be easiest to assess on a case by case basis. Upon confirmation of a vulnerability, we agree to award responsible disclosures at minimum 100k sats and as high as the total potential loss that would result from exploiting the vulnerability.
+
+### Rules
+1. Disclosure is responsible and does not increase the likelihood of an exploit
+2. Disclosure details how to exploit the vulnerability with certainty
```

Comment:
I would write the following:

> ### Rules
> 1. Disclosure is responsible and does not increase the likelihood of an exploit
> 2. A detailed report with reproducible steps is provided. If the report is not detailed enough to reproduce the issue, the issue will not be eligible for a reward.
> 3. A realistic attack scenario including prerequisites for an attack and expected gains after the exploitation is provided. Reports without such scenario, with unrealistic assumptions or without meaningful outcomes will not be eligible for reward.


Inspired by Cloudflare Public Bug Bounty Program Rules:

> Please provide detailed reports with reproducible steps. If the report is not detailed enough to reproduce the issue, the issue will not be eligible for a reward.
> Please provide a realistic attack scenario including prerequisites for an attack and expected gains after the exploitation. Reports without such scenario, with unrealistic assumptions or without meaningful outcomes will not be eligible for reward.

-- https://hackerone.com/cloudflare